### PR TITLE
add github ci badge

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

**Comment by liweinan** [comment]
@jamezp maybe it's good to have Github CI badge too :) :

image

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # RESTEasy
 
-[![Build Status](https://travis-ci.org/resteasy/Resteasy.svg?branch=master)](https://travis-ci.org/resteasy/Resteasy)
+[![Travis Build Status](https://travis-ci.org/resteasy/Resteasy.svg?branch=master)](https://travis-ci.org/resteasy/Resteasy)
```
Comment (jamezp) [inline README.md:3]: I wonder if we should remove this one as we removed the travis.yml due it going offline tomorrow.

Comment (liweinan) [reply README.md:3]: @jamezp okay!

**Comment by liweinan** [comment]
@jamezp old badge removed!

**Review by jamezp — APPROVED** [review]

**Pull request merged**